queue_services/entity-emailer/src/entity_emailer/message_tracker/tracker.py

```diff
@@ -61,6 +61,13 @@ def get_message_context_properties(queue_msg: nats.aio.client.Msg):
             identifier = email_msg.get('identifier', None)
             message_id = f'{etype}_{filing_id}'
             return create_message_context_properties(etype, message_id, None, identifier, False)
+
+        if etype == 'bc.registry.dissolution':
+            message_id = email_msg.get('id', None)
+            identifier = email_msg.get('identifier', None)
+            if involuntary_dissolution_stage_1_notification.is_processable(email_msg):
```

Comment:
As I had mentioned previously, you can extend the existing code above for `if etype == 'bc.registry.names.request':`. Just update the `if` block to use `in` and add `'bc.registry.dissolution'` to it.

And the `if involuntary_dissolution_stage_1_notification.is_processable(email_msg):` check is not required here.

![image](https://github.com/user-attachments/assets/beaae48c-d6e3-4ab4-8312-a24e37c7e786)

Reply:
Then, check the business.legal_type at the same place ( check the furnishing_name )?

Reply:
Actually, you are right after thinking a bit more about hits.  

Let's leave this code like you have it but I don't want to have the tracker code calling the processor code.

And we don't need to check that it's a BC company.

Reply:
`we don't need to check that it's a BC company.`
Then, remove this part from the tracker

Reply:
Removed